### PR TITLE
refactor : full e-com site

### PR DESCRIPTION
**Pull request by TejasShekar**
1. removed redundant code
2. added proper classnames

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: Pages/cart.css [hunk 1]
```diff
@@ -1,46 +1,25 @@
...
-    margin: 1rem;
-    justify-content: space-between;
+.card-info p {
+  text-align: center;
 }
 
-.card-horizontal img {
-    max-width: 10rem;
+.cart-summary {
+  width: 300px;
```
Comment (xii-vi) [inline Pages/cart.css:10]: Use relative unit

**Comment by sah-shah7** [comment]
Everything looks ok

File: Pages/cart.html [hunk 2]
```diff
@@ -16,97 +16,124 @@
...
+      rel="stylesheet"
+      href="https://prime8-ui.netlify.app/styles.css"
+    />
     <link rel="stylesheet" href="../home.css" />
     <link rel="stylesheet" href="./cart.css" />
   </head>
   <body>
-    <header class="header">
-      <a href="../index.html" class="brand-name">
+    <header class="header flex-sbw-c flex-wrap">
```
Comment (xii-vi) [inline Pages/cart.html:27]: what is sbw ?

**Review by xii-vi — APPROVED** [review]
_Go through comments before merging._

File: Pages/cart.css (hunk 1 above)
Comment (TejasShekar) [reply Pages/cart.css:10]: done

**Review by sah-shah7 — APPROVED** [review]

File: Pages/cart.html (hunk 2 above)
Comment (TejasShekar) [reply Pages/cart.html:27]: its space-between.

**Pull request merged**